source/common.ts

```diff
@@ -474,19 +259,33 @@ const i18n = {
 i18n.__ = i18n.getTranslation;
 i18n.addTranslation = i18n.addTranslations;
 
-function format(integer: number, separator: string) {
-  let result = '';
-  while (integer) {
-    const n = integer % 1e3;
-    integer = Math.floor(integer / 1e3);
-    if (integer === 0) {
-      return n + result;
+function interpolateTranslation(
+  variables: Omit<GetTranslationOptions, "_locale" | "_purify">,
```

Comment:
`_purify` was removed in my previous changes. `Omit<GetTranslationOptions, "_locale">` will be fine.